app-api/src/main/java/org/openmrs/eip/app/logger/AppenderNotifier.java

```diff
@@ -0,0 +1,22 @@
+package org.openmrs.eip.app.logger;
+
+import org.openmrs.eip.app.logger.observer.EventManager;
+import org.openmrs.eip.app.sender.InvalidBinlogOffsetPositionAutoRecovery;
+
+import ch.qos.logback.classic.spi.ILoggingEvent;
+
+public class AppenderNotifier {
```

Comment:
Why is this necessary? Please document some of the new classes

Reply:
Please see [this comment](https://github.com/FriendsInGlobalHealth/openmrs-eip/pull/21#discussion_r1023580970) and share your opinion.

Reply:
Removed while refactoring the code.